ui/js/app.js

```diff
@@ -302,10 +313,10 @@ function chooseCoin(coins, twoWayMode) {
   }
 
   isTwoWay = twoWayMode;
-  setChooseCoinColors();
+  setChooseCoinColors
   // setupAnimation(twoWayMode, aspectRatio800)
 
-  var defaultCoin = coins[0];
+  ();var defaultCoin = coins[0];
```

Comment:
is this on purpose?

Reply:
As we spoke the other day, it's an auto-generated file. Going to resolve all convos related to this file